addons/mail/static/tests/helpers/test_utils.js

```diff
@@ -188,7 +188,12 @@ const VIEW_TYPE_TO_ARCH = {
                      * @returns {integer[]|integer} array of ids if more than one value was passed,
                      * id of created record otherwise.
                      */
-                    create(values = {}) {
+                    create(values) {
```

Comment:
documentation of parameter needs to be adapted to change too

Reply:
@seb-odoo I don't think there is any need to change this documentation of parameters.